cds-snc/cppd-infrastructure
# Azure file share

# Description

Adds the file share in Azure, along with restricting access to only the app service.

# Test Instructions

1. Deploy the infrastructure (including the firewall rules)
2. Push the app container to the registry
3. Verify the connection strings to the database, redis, and storage account
4. Upload a document in the app
5. You can verify that it has been uploaded by viewing the blob storage in the Azure portal.
